package.json

```diff
@@ -34,15 +34,17 @@
   ],
   "contributors": [
     {
-      "name": "Tim Levett",
```

Comment:
I wouldn't remove acknowledging @timlevett . He's an appreciated historical contributor. If there's an emeritus status flag or the like, I think that'd be appropriate, but otherwise I'd just leave him in the `contributors` array, figuring retaining him doesn't harm anything.